### PR TITLE
Added docs vhost to web02

### PR DESCRIPTION
**Pull request by lzap**
As a puppet hater, I have to say this is where it pays off.

### CODE REVIEW

**Comment by lzap** [comment]
Oh right, rebased. I don't see any tests?!

**Comment by ohadlevy** [comment]
I've created docs.tf.org dns entry, so once merged this should be live.

**Comment by mmoll** [comment]
let's try this...

**Comment by mmoll** [comment]
merged, díky @lzap!

File: puppet/modules/web/manifests/init.pp [hunk 1]
```diff
@@ -129,6 +131,24 @@
...
   }
 
+  # DOCS
+  $docs_directory = '/var/www/vhosts/docs/htdocs'
+  $docs_directory_config = [
+    {
+      path            => $docs_directory,
+      options         => ['Indexes', 'FollowSymLinks', 'MultiViews'],
+      expires_active  => 'on',
+      expires_default => 'access plus 4 hours',
```
Comment (ekohl) [inline puppet/modules/web/manifests/init.pp]: Is it wise to set caching headers? This depends on the build process and probably safer to start without.

File: puppet/modules/web/manifests/init.pp [hunk 2]
```diff
@@ -129,6 +131,24 @@
...
+    },
+  ]
+
+  $docs_attrs = {
+    servername   => 'docs.theforeman.org',
+    docroot      => $docs_directory,
+    docroot_mode => '2575',
```
Comment (ekohl) [inline puppet/modules/web/manifests/init.pp]: That's a weird mode. Why is it writable for the group but not the user? We have a special case with yum, but docs should be more like `website`. I'd also prefer a specific user that owns the docroot so we don't have to abuse root for it.

File: puppet/modules/web/manifests/init.pp (hunk 1 above)
Comment (lzap) [reply puppet/modules/web/manifests/init.pp]: I copy-pasted from the yum example, do you want me to remove it?

File: puppet/modules/web/manifests/init.pp (hunk 2 above)
Comment (lzap) [reply puppet/modules/web/manifests/init.pp]: Yeah I picked a really special example. Let me fix it in a new PR.

**Comment by lzap** [comment]
Filed https://github.com/theforeman/foreman-infra/pull/1202

**Pull request merged**